docs/SRS/SRS.tex

```diff
@@ -190,10 +191,16 @@ \subsection{The Purpose of the Project}
 \subsection{The Stakeholders}
 
 \subsubsection{The Client}
+The client of the project is Nabeel Ibrahim. Nabeel will be the point of contact throughout the
+development of the project.
 
 \subsubsection{The Customers}
+The customers of the project will be auto shop owners, shop employees and vehicle owners who are
```

Comment:
Can we change "the project" to Sayyara under customers. Can we change auto shop owners to "independent auto repair shop".